boards/arc/nsim/support/nsim_hs6x.props

```diff
@@ -0,0 +1,31 @@
+	maxlastpc=0
+	trace_enabled=0
+	nsim_isa_family=arc64
+	nsim_isa_core=1
+	arcver=113
+	nsim_isa_atomic_option=1
+	nsim_isa_shift_option=3
+	nsim_isa_code_density_option=2
+	nsim_isa_div_rem_option=2
+	nsim_isa_swap_option=1
+	nsim_isa_bitscan_option=1
+	nsim_isa_enable_timer_0=1
+	nsim_isa_enable_timer_1=1
+	nsim_isa_has_interrupts=1
+	nsim_isa_dual_issue_option=1
+	nsim_isa_div64_option=1
+	nsim_isa_mpy64=1
+	nsim_isa_mpy_option=9
+	nsim_hostlink=1
+	gverbose=0
```

Comment:
I guess in Zephyr we don't need either `nsim_hostlink` or `gverbose` so let's dump it.

Reply:
Done, thanks!